assets/src/scripts/monitoring-location/components/hydrograph/method-picker.test.js

```diff
@@ -141,5 +141,34 @@ describe('monitoring-location/components/hydrograph/method-picker', () => {
 
             expect(div.select('#no-data-points-note').size()).toBe(0);
         });
+
+        it('Expects that if there is not method description, the method id will be used', () => {
```

Comment:
I think the more appropriate place for this test is in time-series-data. After all that is where the logic is. So I recommend this test essentially be moved to time-series-data.test.js

Reply:
Sounds reasonable. Done!